ratpack-session/src/main/java/ratpack/session/internal/ClassWithStaticInit.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ratpack.session.internal;
+
+public class ClassWithStaticInit {
+
+  static {
+    System.out.println("HERE!!!!!!!!!!!!!!!!!!!");
```

Comment:
This seems like a debugging class that shouldn't have been comimtted?

Reply:
What gave it away?

Will remove.